src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -1414,15 +1414,31 @@ profile_event 'cleanup done'
 echo "#"
 echo "# -------------------------------------------------------------------"
 
-echo
-echo "# -------------------------------------------------------------------"
-echo "#"
-echo "# Tarring profiles ..."
-PROFILES_TARBALL="profiles.tgz"
-tar -czf $PROFILES_TARBALL *.prof
-ls -l $PROFILES_TARBALL
-echo "#"
-echo "# -------------------------------------------------------------------"
+if ! test -z "`ls *.prof 2>/dev/null`"
```

Comment:
No, I don't think so -- at least not in a way that using the return val would make a difference.  Like, when max command line length is hit we are screwed either way...